src/moclojer/mrouter.clj

```diff
@@ -0,0 +1,18 @@
+(ns moclojer.mrouter
+  (:require [moclojer.openapi :as openapi]
+            [moclojer.handler :as handler]))
+
+(defn edn-router
+  [{::keys [config]}]
+  (handler/generate-pedestal-route-from-edn config))
+
+(defn router
+  [{::keys [config]}]
+  (if (= (get config "openapi") "3.0.0")
+    (openapi/generate-pedestal-route config)
+    (handler/generate-pedestal-route config)))
+
+(defn smart-router [specs]
```

Comment:
I missed a specific test for testing the `smart-router` function, passing it all possible variations and validating the return

I know we are doing this in isolation by uploading the http server, but I think it is important to have unit tests for this function, so we can put other types of formats in the future

Reply:
agree, I will do some tests for this.